isvcs/container.go

```diff
@@ -769,41 +769,45 @@ func (svc *IService) startupHealthcheck() <-chan error {
 	go func() {
 		var result error
 		if len(svc.HealthChecks) > 0 {
-			checkDefinition, found := svc.HealthChecks[DEFAULT_HEALTHCHECK_NAME]
-			if !found {
-				log.WithFields(logrus.Fields{
-					"healthcheck": DEFAULT_HEALTHCHECK_NAME,
-					"isvc":        svc.Name,
-				}).Debug("Default health check not found")
-				err <- nil
-				return
-			}
 
 			startCheck := time.Now()
+			breakCheck := false
+
 			for {
 				currentTime := time.Now()
 				elapsed := time.Since(startCheck)
-				result = svc.runCheckOrTimeout(checkDefinition)
-				svc.setHealthStatus(result, currentTime.Unix())
-				log := log.WithFields(logrus.Fields{
-					"isvc":    svc.Name,
-					"elapsed": elapsed,
-				})
-				if result == nil {
-					log.Info("Internal service checked in healthy")
-					break
-				} else if elapsed.Seconds() > svc.StartupTimeout.Seconds() {
-					log.WithFields(logrus.Fields{
-						"lastresult": result,
-					}).Warn("Unable to verify health of internal service")
+
+				for healthCheckIndex, healthCheckDefinition := range svc.HealthChecks {
+					result = svc.runCheckOrTimeout(healthCheckDefinition)
+					svc.setHealthStatus(result, currentTime.Unix(), healthCheckIndex)
+
+					log := log.WithFields(logrus.Fields{
+						"isvc":    svc.Name,
+						"elapsed": elapsed,
+					})
+
+					if result == nil {
```

Comment:
I don't think the `Info` message here is necessary since a similar message is logged below in `setHealthStatus`